tests/unit/src/test/scala/tests/BspBuildChangedSuite.scala

```diff
@@ -1,61 +0,0 @@
-package tests
-
-import scala.concurrent.Promise
-
-import scala.meta.internal.metals.MetalsEnrichments._
-
-import bill.Bill
-
-class BspBuildChangedSuite extends BaseLspSuite("bsp-build-changed") {
```

Comment:
So this suite was testing exactly what I removed. I think the Scala CLI test makes much more sense.